.eslintrc.json

```diff
@@ -15,7 +15,7 @@
         "react"
     ],
     "rules": {
-        "class-methods-use-this": "warn",
+        "class-methods-use-this": ["error", {"exceptMethods": ["render"]}],
```

Comment:
does this exclude a method from this rule? if so, why?

Reply:
good question! `render` is a react lifecycle method that can't be made static.